dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/PoolTaskRestRepository.java

```diff
@@ -161,4 +133,52 @@ public void afterPropertiesSet() throws Exception {
                 new Link("/api/" + PoolTaskRest.CATEGORY + "/" + PoolTaskRest.NAME + "/search",
                         PoolTaskRest.NAME + "-search")));
     }
+
+    @PreAuthorize("hasAuthority('ADMIN')")
+    @SearchRestMethod(name = "findAllByItem")
+    public Page<PoolTaskRest> findAllByItem(@Parameter(value = "uuid", required = true) UUID itemUUID,
+           Pageable pageable) {
+        List<PoolTask> poolTasks = null;
+        try {
+            Context context = obtainContext();
+            Item item = itemService.find(context, itemUUID);
+            if (item == null) {
+                throw new UnprocessableEntityException("There is no Item with uuid provided, uuid:" + itemUUID);
+            }
+            XmlWorkflowItem xmlWorkflowItem = xmlWorkflowItemService.findByItem(context, item);
+            if (xmlWorkflowItem == null) {
+                return null;
```

Comment:
I'm adding this reference more for me to avoid to look back to that again and again :) 
https://github.com/DSpace/DSpace/pull/3054/commits/3b5130c5a41d96ccf485b44a6a4d844b124261d3#diff-fc493b8186aa975a2deded8f3c737ceffaa076fad1161fe0c3d7080246237bcdR3924
and next test

Reply:
@abollini this IT verifies /api/workflow/claimedtask/search/findByItem instead of /api/workflow/pooltasks/search/findAllByItem

Specifically for /api/workflow/pooltasks/search/findAllByItem I didn't see a method which tests for an existing item which is not in the workflow

Reply:
@benbosman thanks for the feedback, the test has been added: https://github.com/DSpace/DSpace/pull/3054/commits/cfa8b0348bd8d5d824102b8a7b75c6bc0a86e2a4#diff-fc493b8186aa975a2deded8f3c737ceffaa076fad1161fe0c3d7080246237bcdR3489